pkg/cli/managercli/run.go

```diff
@@ -39,7 +46,15 @@ func RunCmd() *cobra.Command {
 			}
 
 			// Create a new Cmd to provide shared dependencies and start components
-			mgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: v.GetString("metrics-addr")})
+			options := manager.Options{
+				MetricsBindAddress: v.GetString("metrics-addr"),
+			}
+
+			if v.GetString("namespace") != "" {
+				options.Namespace = v.GetString("namespace")
```

Comment:
This causes the Watch that's started in the reconcile loop to only require a role/rolebinding instead of the default clusterrole/clusterrolebinding as changed in: kubernetes-sigs/controller-runtime#124